lib/api/formtools/grid.js

```diff
@@ -0,0 +1,263 @@
+var linz = require('../../../');
+    async = require('async'),
+    clone = require('clone');
+
+(function  () {
+
+    function getSettings(modelName, cb) {
+
+        if (!modelName || typeof(modelName) !== 'string' || !modelName.length) {
+            throw new Error('modelName is required and must be of type string and cannot be emptied.');
+        }
+
+        var model = linz.get('models')[modelName];
+
+        model.getGrid(function (err, settings) {
+
+            if (err) {
+                return cb(err);
+            }
+
+            return cb(err, clone(settings));
+
+        });
+
+    }
+
+    function getToolbarItems (req, res, modelName, cb) {
```

Comment:
I think we should both install https://atom.io/packages/docblockr and make sure that we use this. I think this code would benefit from some basic explanations of the functions. What do you think?